src/App.vue

```diff
@@ -143,6 +143,10 @@ import {
   Routes
 } from '@/enums'
 import { SessionStorageKeys } from 'sbc-common-components/src/util/constants'
+import { useBusinessStore } from '@/stores/businessStore'
+import { useConfigurationStore } from '@/stores/configurationStore'
+import { useFilingHistoryListStore } from '@/stores/filingHistoryListStore'
+import { useRootStore } from '@/stores/rootStore'
```

Comment:
Would it be possible to have an index.ts file in `@/stores/` and then `import * from '@/stores/` here?

Reply:
The index file should work but I think the import won't.

I was thinking of this: https://github.com/bcgov/business-create-ui/blob/4fdd677f468632477e027e8da840ca539158aa39/src/App.vue#L207

Which can then be used like this: https://github.com/bcgov/business-create-ui/blob/4fdd677f468632477e027e8da840ca539158aa39/src/App.vue#L235

I think "import *" doesn't buy us anything, but this would be cleaner at least:
```
import { useBusinessStore, useConfigurationStore, useFilingHistoryListStore, useRootStore } from `@/stores/`
```

Reply:
I like this a lot! I'll be doing this.

Reply:
A question regarding this, I already have an index.ts file which exports both the Vuex and Pinia stores. Is it OK to add exports for all the individual Pinia stores in that file as well?

Reply:
I don't see why not. The "importer" can specific which exported things it imports, so it won't import the things it doesn't need.

Reply:
Fixed!